lib/expression/eval.ex

```diff
@@ -198,6 +198,33 @@ defmodule Expression.Eval do
   def op(:=, a, b) when is_struct(a, Date) and is_struct(b, Date),
     do: Date.compare(a, b) == :eq
 
+  # credo:disable-for-next-line Credo.Check.Refactor.CyclomaticComplexity
+  def op(operator, a, b)
+      when operator in [:=, :==, :!=, :<, :<=, :>, :>=] and
+             (is_struct(a, Date) or is_struct(a, DateTime)) and
+             is_binary(b) do
+    comparison_result =
+      if is_struct(a, Date) do
+        Date.compare(a, Expression.DateHelpers.extract_dateish(b))
+      else
+        DateTime.compare(a, Expression.DateHelpers.extract_datetimeish(b))
+      end
+
+    case {operator, comparison_result} do
+      {:=, :eq} -> true
```

Comment:
are we intentionally supporting the _match_ operator as an _equal_ operator?

Reply:
yes, this was already the case in the existing code. The FLOIP standard supports both `=` and `==` (or at least RapidPro does).